packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -246,10 +346,14 @@ const Component: React.FC<Props> = (props) => {
         props.onClickCreate(theCondition)
     }
 
-    const isOperatorDisabled = selectedEntityOption?.data.entityType === CLM.EntityType.ENUM
+    let isOperatorDisabled = (selectedEntityOption?.data.entityType === CLM.EntityType.ENUM) || (selectedComparisonType === stringComparisonTypeOption)
+
     const conditionsUsingEntity = props.conditions.filter(c => c.entityId === selectedEntityOption?.key)
     const currentCondition = createConditionFromState()
 
+
+    let isComparisonTypeDisabled = (selectedEntityOption?.data.entityType === CLM.EntityType.ENUM)
```

Comment:
seems like it should be `const`.  We only use `let` for things that need to be reassigned later. Same for other `let`s

Reply:
changed